.storybook/lucid-docs-addon/PanelToggles.js

```diff
@@ -1,34 +1,39 @@
 import React from 'react';
 import Hoverable from './Hoverable';
+import addons, { makeDecorator } from '@storybook/addons';
+import ReactDOM from 'react-dom';
 
-export default class extends React.Component {
-	constructor(...args) {
-		super(...args);
-		this.onToggleLayout = this.onToggleLayout.bind(this);
-		this.onTogglePanel = this.onTogglePanel.bind(this);
-	}
-
-	onToggleLayout() {
-		const urlState = this.props.api.getUrlState();
-		this.props.api.setOptions({
-			addonPanelInRight: !urlState.panelRight,
-		});
-	}
+const portalElement = window.document.createElement('div');
+window.parent.document.body.appendChild(portalElement);
```

Comment:
storybook puts content in an iframe, so the statically positioned toggle would be pinned to the iframe.  We shouldn't run into any cross-origin issues.